templates/reporting/user_list_period_data.html.twig

```diff
@@ -39,7 +39,7 @@
         {% set usersTotalInternalRate = 0 %}
         {% set usersTotalRate = 0 %}
         {% for period in attribute(userPeriod, period_attribute) %}
-            {% if period.totalDuration > 0 %}
+            {% if period.totalDuration != 0 %}
```

Comment:
If I compare with the other template, we can remove the 3 x  `if != 0` / `if > 0` checks completely:
https://github.com/kimai/kimai/blob/main/templates/reporting/report_by_user_data.html.twig#L56

Reply:
I'm not sure I fully understand what you mean. Something like this?

```
{% for period in attribute(userPeriod, period_attribute) %}
    {% set usersTotalDuration = usersTotalDuration + period.totalDuration %}
    {% set absoluteDuration = absoluteDuration + period.totalDuration %}
    {% set usersTotalInternalRate = usersTotalInternalRate + period.totalInternalRate %}
    {% set absoluteInternalRate = absoluteInternalRate + period.totalInternalRate %}
    {% set usersTotalRate = usersTotalRate + period.totalRate %}
    {% set absoluteRate = absoluteRate + period.totalRate %}
    {% set reportDateKey = period.date|report_date %}
    {% set totalsDuration = totalsDuration|merge({(reportDateKey): (totalsDuration[reportDateKey] + period.totalDuration)}) %}
    {% set totalsInternalRate = totalsInternalRate|merge({(reportDateKey): (totalsInternalRate[reportDateKey] + period.totalInternalRate)}) %}
    {% set totalsRate = totalsRate|merge({(reportDateKey): (totalsRate[reportDateKey] + period.totalRate)}) %}
{% endfor %}
```

Reply:
Yes exactly that!